doc/source/examples/multiphysics/laser-melting/laser-melting.rst

```diff
@@ -0,0 +1,287 @@
+    	set type	      = convection-radiation
+            set h	      	      = 15
+            set Tinf	      = 20
+            set emissivity        = 0.5
+        end
+        subsection bc 1
+        	set id = 1
+    	set type	      = convection-radiation
+            set h	              = 15
+            set Tinf	      = 20
+            set emissivity        = 0.5
+        end
+        subsection bc 2
+        	set id = 2
+    	set type	      = convection-radiation
+            set h	              = 15
+            set Tinf	      = 20
+            set emissivity        = 0.5
+        end
+        subsection bc 3
+        	set id = 3
+    	set type	      = convection-radiation
+            set h	              = 15
+            set Tinf	      = 20
+            set emissivity        = 0.5
+        end
+    end
+
+
+The ``multiphysics`` subsection enables to turn on `(true)` 
+and off `(false)` the physics of interest. Here ``heat transfer`` is enabled.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set heat transfer  		= true
+    end 
+    
+
+In the ``laser parameters`` section, the parameters of the laser model are defined. The exponential decaying model [1] is used to simulate the laser heat source. In the exponential decaying model, the laser heat flux is calculated using the following equation:
+
+    .. math:: 
+        q(x,y,z) = \frac{\eta \alpha P}{\pi r^2 \mu} \exp{(-\eta \frac{r^2}{R^2})} \exp{(- \frac{|z|}{\mu})}
+
+
+where :math:`\eta`, :math:`\alpha`, :math:`P`, :math:`R`, :math:`\mu`, :math:`r`, \:math:`z` denote concentration factor, absorptivity, laser power, beam radius, penetration depth, radial distance from the laser focal point, and axial distance from the laser focal point, respectively.
+
+
+.. note:: 
+    The scanning path of the laser is defined using a Function exptression in the ``path`` subsection.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Laser parameters
+    #---------------------------------------------------
+    subsection laser parameters
+    	set enable = true
+        	set concentration factor 	= 2
+        	set power 			= 100
+        	set absorptivity 		= 0.6
+        	set penetration depth 		= 0.000070
+        	set beam radius 		= 0.000050
+        	set start time 			= 0
+        	set end time 			= 0.0034
+        	set beam orientation 		= y
+        	subsection path
+        		set Function expression =  0.5 * t; 0.000500
+        	end
+    end    
+
+
+The laser heat source locally melts the material, which is initially in the solid phase according to the definition of the ``solidus temperature``. Hence, the physical properties should be defined using ``phase_change`` models. In the ``physical properties`` subsection, the physical properties of the different phases of the fluid are defined:
```

Comment:
Maybe point out to the Stefan example for a more thorough explanation of this?

Reply:
done.